docs/styles.css

```diff
@@ -123,21 +131,29 @@ a:active {
   list-style-type: none;
   margin: 0;
   padding: 0;
-  border: 1px solid #ddd;
+  padding-bottom: 0.25rem;
+  border: var(--border);
+  box-shadow: var(--box-shadow);
 }
 
-#category-search-results li {
-  padding: 2px 4px;
+.category-search-container li {
```

Comment:
not really important for an app this small, but probably better practice to attach styles to a classname rather than an ID 🤷